react/searchResult.css

```diff
@@ -177,6 +177,18 @@
 
 .filterIcon {}
 
+.sortByBt {}
```

Comment:
why "sort" instead of "order"? (I think sort might actually be better, but "order" is used elsewhere and I think it might be better to keep it for consistency)

Also refrain from abbreviating names unless it'd be extremely long otherwise ("orderByButton" would be just fine)

Reply:
Agree with this change.